### ISSUE TITLE
Thank you very much

### ISSUE THREAD

**Issue by huseinzol05** [xueyouluo/my_seq2seq#8]
This is not an actual issue, I just want to thank you very much, such a great repository, and I did fix some bugs for deprecated issue, but that just around less than 1% what you already done. Also I added Luong Attention for your pointer generator model.

I tested on real dataset, X = body of news, Y = title of news. Accuracy is based on sequential cross-entropy, never tested using rouge. All this after 10 epochs, 80% to train 20% to test.

On bahdanau attention, https://github.com/huseinzol05/NLP-Models-Tensorflow/blob/master/abstractive-summarization/7.xueyouluo-pointer-generator-bahdanau.ipynb
```
epoch: 9, avg loss: 6.818628, avg accuracy: 0.913240
epoch: 9, avg loss test: 16.073720, avg accuracy test: 0.870706
```

On luong attention, https://github.com/huseinzol05/NLP-Models-Tensorflow/blob/master/abstractive-summarization/9.xueyouluo-pointer-generator-luong.ipynb
```
epoch: 9, avg loss: 4.113721, avg accuracy: 0.988244
epoch: 9, avg loss test: 14.028113, avg accuracy test: 0.915375
```

Thank you so much!

**Comment by xueyouluo** [comment]
@huseinzol05 Great work! I have looked your repo,  really great work! Stared! I didn't maintain this repo for a long time, sorry for the buggy code😂
